README.md

```diff
@@ -164,6 +164,19 @@ will still be generated.
 - [`web-sys`][web-sys] for bindings to Web APIs like `window.fetch`, WebGL,
   WebAudio, etc. (*optional*, via feature-flag)
 
+### Generating GitHub Actions, Docker, Wasm workspace, etc in an existing project
+
+The `rust` and `rust+wasm` templates are also designed for specific prompts
+when working within existing Rust projects when using the [`--init` flag][cargo-generate-init].
```

Comment:
Having trouble parsing this statement. Is it the generator or the templates themselves that have specific prompts? I think it's the former.

Also, I'm not sure where I should run the command from reading this description. I think it should be run in the project root.

So maybe this could be written with something like:

> The generator is designed to add to an existing project and prompts with this in mind. To generate in an existing project, run this command in the project root: